apps/challenges/views.py

```diff
@@ -2997,6 +2998,16 @@ def manage_worker(request, challenge_pk, action):
 
     challenge = get_challenge_model(challenge_pk)
 
+    if datetime.fromisoformat(
```

Comment:
Can you instead use the datetime parser? See this https://dateutil.readthedocs.io/en/stable/parser.html